pandas/tests/computation/test_eval.py

```diff
@@ -608,6 +608,12 @@ def test_unary_in_array(self):
                       -False, False, ~False, +False,
                       -37, 37, ~37, +37], dtype=np.object_))
 
+    def test_float_comparison_bin_op(self):
+        # GH 16363
+        df = pd.DataFrame({'x': np.array([0], dtype=np.float32)})
+        res = df.eval('x < -0.1')
+        assert np.array_equal(res, np.array([False])), res
```

Comment:
What's the reason for `, res` here? I think also causing CI failure

Reply:
Good catch, removing this.